rest/client/mock.go

```diff
@@ -23,6 +24,7 @@ import (
 	"github.com/mongodb/grip"
 	"github.com/mongodb/grip/message"
 	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2/bson"
```

Comment:
we need to not depend on the bson package outside of model/ and children, I think.